plugins/libplugin.h

```diff
@@ -468,6 +468,7 @@ struct listpeers_channel {
 	struct node_id id;
 	bool connected;
 	bool private;
+	bool lost_state;
```

Comment:
Don't reuse this code, as it's going away eventually (sorry!)

Reply:
Okay, I'll parse manually.